internal/providers/terraform/aws/aws_ecs_service.go

```diff
@@ -0,0 +1,90 @@
+package aws
+
+import (
+	"fmt"
+	"infracost/pkg/schema"
+	"regexp"
+	"strings"
+
+	"github.com/shopspring/decimal"
+)
+
+func AwsEcsService(d *schema.ResourceData) *schema.Resource {
+	launchType := d.Get("launch_type").String()
+	if launchType != "FARGATE" {
+		return nil
```

Comment:
just FYI you also have the option to use a price label override on a PriceComponent

```go
priceComponent.SetPriceOverrideLabel("coming soon")
```

Reply:
Agree we need some way of logging these. It gets more complicated when we think about logging all resources we don't support prices for. We'd need to go through every resource and check if it has any prices or not, because we don't want to be logging warnings for resources that have no costs anyway.

Reply:
I think for the resources we do not support at all, we can skip the whole process. But there are resources which we partially support. In such cases, we can warn or inform the user about why we did not calculate the price